galaxy_importer/loaders.py

```diff
@@ -106,14 +141,15 @@ def _get_doc_strings(self):
 
         if not json_output:
             return
-        data = json.loads(json_output)
-        if self.name not in data.keys():
-            self.log.error(f'No "{self.name}" key in ansible-doc output')
-            return None
 
+        data = json.loads(json_output)
+        if len(data.keys()) != 1:
```

Comment:
In theory, data could be a `list` without a .keys() method. Hopefully ansible-doc output won't change (again?) though. Is there a unit test where the ansible-doc output is invalid (a list, etc?). Assuming the errors/traceback bubble back up in a useful way to detect it, I wouldn't worry about type checking, but a unit test that expects an exception may be useful.

Reply:
added a type check, since the traceback was not super helpful